### PR TITLE
Add support for XML files through xmltodict library

### PR DESCRIPTION
**Pull request by DarkWizarD24**
Add support for XML file as input. Work like the YAML support.

### CODE REVIEW

File: jinja2cli/cli.py [hunk 1]
```diff
@@ -194,6 +211,8 @@ def cli(opts, args):
             else:
                 if ext in ('yml', 'yaml'):
                     raise InvalidDataFormat('%s: install pyyaml to fix' % ext)
+                if ext in ('xml'):
```
Comment (mattrobenolt) [inline jinja2cli/cli.py]: Can we just use `==` here?

File: jinja2cli/cli.py [hunk 2]
```diff
@@ -282,6 +301,8 @@ def main():
     if opts.format not in formats and opts.format != 'auto':
         if opts.format in ('yml', 'yaml'):
             raise InvalidDataFormat('%s: install pyyaml to fix' % opts.format)
+        if opts.format in ('xml'):
```
Comment (mattrobenolt) [inline jinja2cli/cli.py]: Can we just use `==` here?

**Comment by mattrobenolt** [comment]
This lgtm, but I really want to do #17 before accepting this. With all of these formats, it's starting to impact startup time. :)

**Comment by mattrobenolt** [comment]
I would also add this under `extras` inside `setup.py` so we could do `$ pip install jinja2-cli[xml]`

**Comment by DarkWizarD24** [comment]
You are right, https://github.com/mattrobenolt/jinja2-cli/issues/17 need to be fixed before adding xml support.

**Comment by mattrobenolt** [comment]
Resolved with #29. Thanks. :)

**Pull request closed without merging**